shared/idmapset.go

```diff
@@ -158,7 +158,7 @@ func (m IdmapSet) Intersects(i idmapEntry) bool {
 func (m IdmapSet) ToLxcString() []string {
 	var lines []string
 	for _, e := range m.idmap {
-		lines = append(lines, e.ToLxcString()+"\n")
+		lines = append(lines, e.ToLxcString())
```

Comment:
Don't do that, it's going to break container_new as the LXC config will be invalid.

The \n problem has already been solved in master so you can just drop this change.